medication statements/README.md

```diff
@@ -0,0 +1,363 @@
+4. If `medicationRequest.dispenseRequest.validityPeriod.end` does not exist then this value is not populated
+5. When `medicationRequest.extension.value.coding[0].code` = `"acute"` or `"acute-handwritten"` and `medicationRequest.extension.url` = `"https://fhir.nhs.uk/STU3/StructureDefinition/Extension-CareConnect-GPC-PrescriptionType-1"` then value used is `0`.</br> When `medicationRequest.extension.value.coding[0].code` = `"delayed-prescribing"`, `"repeat"` or `"repeat-dispensing"` then `medicationRequest.extension[0].value` when `medicationRequest.extension[0].url` = `"numberOfRepeatPrescriptionsAllowed"`.</br> If no value is found in either case then a default value of `1` is used.
+6. When `medicationRequest.dispenseRequest.extension[0].url` = `"https://fhir.nhs.uk/STU3/StructureDefinition/Extension-CareConnect-GPC-MedicationQuantityText-1"`
+7. When 'medicationRequest.dispenseRequest.quantity.extension[0].url' = `"https://fhir.nhs.uk/STU3/StructureDefinition/Extension-CareConnect-GPC-MedicationQuantityText-1"`
+8. When no other values are found a default value of `"1"` is used.
+
+
+### EhrSupplyDiscontinue mapping
+
+This is part of the mapping for a `MedicationRequest` and is mapped when one of `MedicationRequest.extension.url` = `"https://fhir.nhs.uk/STU3/StructureDefinition/Extension-CareConnect-GPC-MedicationStatusReason-1"` (Medication Status Reason Stopped URL).  If this is not present then this mapping is omitted.
+
+
+| Mapped to (EhrSupplyDiscontinue XML HL7)                                                                           | Mapped from (JSON FHIR / other source )                                                                                                                                                                                                                 |
+|--------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| ehrSupplyDiscontinue / id \[@root]                                                                                 | adaptor generated UUID                                                                                                                                                                                                                                  |
+| ehrSupplyDiscontinue / code                                                                                        | set from the `codableConcept` in `medicationRequest.extension[0].extension[0].value`<sup>1</sup><sup>2</sup>                                                                                                                                            |
```

Comment:
I think the double extension here adds confusion, perhaps add a definition to the top for the first extension for "MedicationStatusReason".

The second level of extension here, looks like it's also filtering on "statusReason".

Reply:
have done as suggested